<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="agi-lt-882" ignored="0" name="Default test" tests="6" failures="1" timestamp="06 Feb 2021 04:06:21 GMT" time="246.492" errors="0">
  <testcase name="testLaunchUrl" time="0.362" classname="Scenario_Component.LaunchUrl"/>
  <testcase name="testLoginLogout" time="62.523" classname="Scenario_Component.LaunchUrl"/>
  <testcase name="testAerRegistration" time="30.383" classname="Scenario_Component.LaunchUrl"/>
  <testcase name="testCaseIntake" time="20.261" classname="Scenario_Component.LaunchUrl"/>
  <testcase name="testFullDataEntry" time="86.825" classname="Scenario_Component.LaunchUrl"/>
  <testcase name="testAERRegistrationInformation" time="32.085" classname="Scenario_Component.LaunchUrl">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Generic_Component.ExcelReadWrite.writecell(ExcelReadWrite.java:95)
at PageObject_Component.AERRegistrationinformation.getCaseNumber(AERRegistrationinformation.java:58)
at Scenario_Component.LaunchUrl.testAERRegistrationInformation(LaunchUrl.java:150)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testAERRegistrationInformation -->
</testsuite> <!-- Default test -->
